src/interface/thirdparty-api.yaml

```diff
@@ -0,0 +1,2352 @@
+openapi: 3.0.2
+info:
+  version: "1.0"
+  title: Mojaloop ThirdParty API Adapter
+  description:
+    A Mojaloop API for thirdparty interactions between `PISPs` (Payment Initiation
+    Service Providers) and `DFSPs` (Digital Financial Service Providers)
+  license:
+    name: TBD
+    url: TBD
+servers:
+  - url: '{protocol}://hostname:<port>/'
+    variables:
+      protocol:
+        enum:
+          - http
+          - https
+        default: https
+paths:
+  /health:
+    get:
+      operationId: HealthGet
+      summary: Get current status of the API
+      description: The HTTP request `GET /health` is used to return the current status of the API.
+      tags:
+        - health
+      responses:
+        200:
+          $ref: '#/components/responses/200'
+        400:
+          $ref: '#/components/responses/400'
+        401:
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        404:
+          $ref: '#/components/responses/404'
+        405:
+          $ref: '#/components/responses/405'
+        406:
+          $ref: '#/components/responses/406'
+        501:
+          $ref: '#/components/responses/501'
+        503:
+          $ref: '#/components/responses/503'
+  /metrics:
```

Comment:
I suppose this is boilerplate, but we should leave it in as we may want it in the future.

Reply:
/health and /metrics are standard endpoints I've seen in mojaloop microservices

Reply:
OK